pkg/apply/mutator/mutator.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2021 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package mutator
+
+import (
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+// Interface decouples apply-time-mutation
+// from the concrete structs used for applying.
+type Interface interface {
```

Comment:
nit: consider a different name, like "Mutator"

Reply:
Golint complains about stutter on exported functions and types:
- https://github.com/golang/lint/blob/master/lint.go#L483
- https://github.com/golang/lint/blob/master/lint.go#L938

EX: sort.Interface instead of sort.Sort: https://pkg.go.dev/sort#Interface

I think this used to be in Effective Go, but now that I'm looking for it, I can't find it in the latest copy.

It was in older versions tho:
- https://www.rose-hulman.edu/class/csse/csse403/Fall2010/SlidePDFs/EffectiveGo.pdf
- https://www.mit.edu/afs.new/sipb/project/golang/doc/effective_go.html#names